scss/lessonCard.scss

```diff
@@ -0,0 +1,21 @@
+.card-title {
```

Comment:
according to [BEM](https://en.bem.info/methodology/quick-start/), this should be `.lesson-card__title`

Reply:
Corrected 👍